codespell_lib/data/dictionary.txt

```diff
@@ -9622,6 +9671,7 @@ mimimise->minimise, minimize,
 mimimize->minimize, minimise,
 mimimum->minimum
 mimimun->minimum
+miminal->minimal
```

Comment:
Please add `miminally->minimally` and `miminaly->minimally`

Reply:
Done